Apps/Common/SIMCoupled.h

```diff
@@ -53,7 +53,13 @@ template<class T1, class T2> class SIMCoupled
   }
 
   //! \brief Computes the solution for the current time step.
-  virtual bool solveStep(TimeStep& tp, bool firstS1 = true)
+  virtual bool solveStep(TimeStep& tp)
+  {
+    return this->solveStep(tp, true);
+  }
+
+  //! \brief Computes the solution for the current time step.
+  virtual bool solveStep(TimeStep& tp, bool firstS1)
```

Comment:
Guess you mean ambiguous, og ikke "ambiøs", as I read it first.. This was maybe another thing pointed out by the new compiler you are trying out? It is still not clear though, how can it be ambiguous when there is only one solveStep method in this hierarchy? solveStep(tp) and solveStep(tp,true) can only resolve to the same method. What other options do the compiler point out? That would be interesting to see (and would probably help my understanding of it).